### PR TITLE
[OpenVINO-EP] V3.4 update samples

### PR DESCRIPTION
**Pull request by MaajidKhan**
**[OpenVINO-EP] V3.4 update samples:**
-> Updated c# sample
-> Added I/O buffer c++ sample and it's documentation
-> Added Requirements.txt file for tinyYolov2 python sample

### CODE REVIEW

File: c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ The source code for this sample is available [here](https://github.com/microsoft
```
Comment (oliviajain) [inline c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md:10]: Above How to Build change 
The source code for this sample is available [here](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx/OpenVINO_EP/squeezenet_classification).
to 
The source code for this sample is available [here](https://github.com/microsoft/onnxruntime-inference-examples/tree/main/c_cxx/OpenVINO_EP/**Linux/**squeezenet_classification).

File: python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md [hunk 2]
```diff
@@ -29,6 +29,8 @@ The source code for this sample is available [here](https://github.com/microsoft
 * use any sample video with objects as test input to this sample
 * Download the tinyYOLOv2 model from the ONNX Model Zoo
 
+Note: For all the python package dependencies requirements, check 'requirements.txt' file in the sample directory
```
Comment (oliviajain) [inline python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md:32]: Add: You may install these with `pip install -r requirements.txt`.

File: python/OpenVINO_EP/yolov4_object_detection/README.md [hunk 3]
```diff
@@ -36,7 +36,7 @@ The source code for this sample is available [here](https://github.com/microsoft
 * use any sample video with objects as test input to this sample
 * Download the tinyYOLOv2 model from the ONNX Model Zoo
 [Download Sample videos](https://github.com/intel-iot-devkit/sample-videos)
-* For all the python package dependencies requirements, check 'requirements.txt' file in the sample      directory.
+Note: For all the python package dependencies requirements, check 'requirements.txt' file in the sample directory.
```
Comment (oliviajain) [inline python/OpenVINO_EP/yolov4_object_detection/README.md:39]: Add: You may install these with pip install -r requirements.txt.

Comment (MaajidKhan) [reply python/OpenVINO_EP/yolov4_object_detection/README.md]: Done

File: python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md (hunk 2 above)
Comment (MaajidKhan) [reply python/OpenVINO_EP/tiny_yolo_v2_object_detection/README.md]: Done

File: c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md [hunk 4]
```diff
@@ -37,26 +41,43 @@ If you build it by yourself, you must append the "--build_shared_lib" flag to yo
...
+      (using Default CPU)
+      ```
+      ./run_squeezenet --use_cpu <path_to_onnx_model> <path_to_sample_image> <path_to_labels_file>
+      ```
+   - To Run the sample for IO Buffer Optimization feature
+      ```
+      ./run_squeezenet <path_to_onnx_model> <path_to_sample_image> <path_to_labels_file>
+      ```
 
 ## References:
```
[the remark this answers is not in the document]
Comment (MaajidKhan) [reply c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md:83]: Done

File: c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md (hunk 1 above)
Comment (MaajidKhan) [reply c_cxx/OpenVINO_EP/Linux/squeezenet_classification/README.md:10]: Done

**Review by oliviajain — APPROVED** [review]

**Comment by oliviajain** [comment]
@MaajidKhan "This branch is out-of-date with the base branch." I think you may need to merge master in.

**Comment by MaajidKhan** [comment]
> @MaajidKhan "This branch is out-of-date with the base branch." I think you may need to merge master in.

@oliviajain Done Olivia. I rebased this branch with master. This branch is up-to-date now.

**Pull request merged**